### ISSUE TITLE
[FEAT] Expose additional hyperparameters for chemprop (+ensemble) training

### ISSUE THREAD

**Issue by smcolby** [OpenADMET/openadmet-models#480]
Expose ability to set different learning rates and weight decay for MPNN and FFN components, reduce LR on plateau scheduler, and gradient clipping. Enable ensembling with and without bootstrapping.

**Comment by smcolby** [comment]
Resolved by #576.
